src/region.js

```diff
@@ -269,7 +275,11 @@ const Region = MarionetteObject.extend({
     delete this.currentView;
 
     if (!view._isDestroyed) {
-      this._removeView(view, shouldDestroy);
+      if (shouldDestroy) {
+        this.removeView(view);
+      } else {
+        this._detachView(view);
```

Comment:
Its the opposite. As is today, using `_removeView` as is done by http://codepen.io/blikblum/pen/PppGmG, calling `detachView` will animate. With this PR it will not animate which i think is the correct behavior